<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ROM Properties Page Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="ImageTypesTab" name="tabImageTypes">
      <attribute name="title">
       <string>&amp;Image Types</string>
      </attribute>
     </widget>
     <widget class="DownloadsTab" name="tabDownloads">
      <attribute name="title">
       <string>&amp;Downloads</string>
      </attribute>
     </widget>
     <widget class="AboutTab" name="tabAbout">
      <attribute name="title">
       <string>&amp;About</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageTypesTab</class>
   <extends>QWidget</extends>
   <header>config/ImageTypesTab.hpp</header>
   <container>1</container>
   <slots>
    <signal>modified()</signal>
    <slot>reset()</slot>
    <slot>loadDefaults()</slot>
    <slot>save()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>DownloadsTab</class>
   <extends>QWidget</extends>
   <header>config/DownloadsTab.hpp</header>
   <container>1</container>
   <slots>
    <signal>modified()</signal>
    <slot>reset()</slot>
    <slot>loadDefaults()</slot>
    <slot>save()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>AboutTab</class>
   <extends>QWidget</extends>
   <header>config/AboutTab.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tabModified()</slot>
 </slots>
</ui>
